storage/src/factory.rs

```diff
@@ -117,6 +117,7 @@ impl BlobFactory {
     ) -> IOResult<Arc<dyn BlobCache>> {
         let backend_cfg = config.get_backend_config()?;
         let cache_cfg = config.get_cache_config()?;
+        let ondemand_batch_size = config.get_rafs_config()?.batch_size;
```

Comment:
Does this have a default value if we do not set it?

Reply:
I'm not sure when the rafs config is None.
For runtime, it's not empty.
For build time, I didn't observe any failed integration test.
But the unit test `test_block_device()` will fail.
Seems the reason is the config is extracted from `BlobCacheMgr::get_meta_info()`,
https://github.com/dragonflyoss/nydus/blob/3bb124ba77d1deee830807bd48ccb7a9081a368b/service/src/blob_cache.rs#L319-L323
where `BlobCacheEntry` does not store rafs config,
https://github.com/dragonflyoss/nydus/blob/3bb124ba77d1deee830807bd48ccb7a9081a368b/api/src/config.rs#L953-L968
But I don't know when the config extracted from `BlobCacheEntry::get_meta_info()` is used to call `new_blob_cache()`.

Anyway, I changed this code to assume that rafs config can be None here, for passing the unit test `test_block_device()`.

Reply:
`batch_size` does have a default value if it is not set in the config file.
https://github.com/dragonflyoss/nydus/blob/3bb124ba77d1deee830807bd48ccb7a9081a368b/api/src/config.rs#L818-L824